Makefile

```diff
@@ -23,10 +23,12 @@ build: vet test
 	@mkdir -p $(BUILD_DIR)
 	GOOS=linux GOARCH=amd64 $(GO_BUILD_CMD) -o $(BUILD_DIR)/$(BINARY_NAME)-linux-amd64
 	GOOS=darwin GOARCH=amd64 $(GO_BUILD_CMD) -o $(BUILD_DIR)/$(BINARY_NAME)-osx-amd64
+	GOOS=windows GOARCH=amd64 $(GO_BUILD_CMD) -o $(BUILD_DIR)/$(BINARY_NAME)-windows-amd64
```

Comment:
Don't forget the ".exe" extension
$(BUILD_DIR)/$(BINARY_NAME)-windows-amd64.exe

Reply:
Fixed.